### ISSUE TITLE
Add actively support to iPadOS (Safari 13+ with MSE)

### ISSUE THREAD

**Issue by avelad** [google/shaka-player#2210]
According to Safari 13 RN https://developer.apple.com/documentation/safari_release_notes/safari_13_release_notes
`Updated Safari for iPad to support Media Source Extensions.`

Note: Safari 13 support webdrivers for automatization, see: https://webkit.org/blog/9395/webdriver-is-coming-to-safari-in-ios-13/

**Comment by joeyparrish** [comment]
Don't we already detect MSE on iPad on iOS 13?

**Comment by avelad** [comment]
Yes, it is currently detected, but in README there is no support. Perhaps the only thing necessary is to put an indication in the README pointing to what is supported.

**Comment by TheModMaker** [comment]
I've confirmed videos play fine in iOS 13, so we can just update the README.

**Comment by mrlika** [comment]
My tests show that on iOS 13.2 iPad Safari Shaka plays MPEG-DASH fine. But for HLS it uses native HTML5 playback, not MSE. It doesn't depend on segments format (TS, MP4) or `mux.js` library.

I check using `getVariantTracks` that returns single quality when HLS is played natively.

I think that MSE should be the default option because it gives more capabilities like manual video quality selection.

**Comment by mrlika** [comment]
`streaming.useNativeHlsOnSafari` doesn't work and I still getting `src=` playback on iPadOS 13.2:

```html
<!DOCTYPE html>
<html>
  <head>
    <!-- Shaka Player compiled library: -->
    <script src="https://cdnjs.cloudflare.com/ajax/libs/shaka-player/2.5.5/shaka-player.compiled.js"></script>
    <script>
var manifestUri =
    'https://akamai-axtest.akamaized.net/routes/lapd-v1-acceptance/www_c4/Manifest.m3u8';

function initApp() {
  // Install built-in polyfills to patch browser incompatibilities.
  shaka.polyfill.installAll();

  // Check to see if the browser supports the basic APIs Shaka needs.
  if (shaka.Player.isBrowserSupported()) {
    // Everything looks good!
    initPlayer();
  } else {
    // This browser does not have the minimum set of APIs we need.
    alert('Browser not supported!');
  }
}

function initPlayer() {
  // Create a Player instance.
  var video = document.getElementById('video');
  var player = new shaka.Player(video);

  // Attach player to the window to make it easy to access in the JS console.
  window.player = player;

  // Listen for error events.
  player.addEventListener('error', onErrorEvent);

  player.configure('streaming.useNativeHlsOnSafari', false);

  // Try to load a manifest.
  // This is an asynchronous process.
  player.load(manifestUri).then(function() {
    // This runs if the asynchronous load is successful.
    alert(player.getVariantTracks().length);
  }).catch(onError);  // onError is executed if the asynchronous load fails.
}

function onErrorEvent(event) {
  // Extract the shaka.util.Error object from the event.
  onError(event.detail);
}

function onError(error) {
  // Log the error.
  console.error('Error code', error.code, 'object', error);
}

document.addEventListener('DOMContentLoaded', initApp);
    </script>
  </head>
  <body>
    <video id="video"
           width="640"
           poster="//shaka-player-demo.appspot.com/assets/poster.jpg"
           controls autoplay></video>
  </body>
</html>

```

**Comment by mrlika** [comment]
Ok. Looks like it is not released in 2.5.5
